content/knowledge_base/flare/consensus/avalanche/concepts.md

```diff
@@ -0,0 +1,30 @@
+# Avalanche Consensus Algorithm Concepts
+
+## Snow Algorithm Group of Protocols
+
+A group of protocols with a strong probabilistic safety guarantee in the presence of Byzantine nodes.
+
+### Description
+
+The core concept of operating is execution of repeated sampling the network at random, ultimately leading to correct nodes behavior to obtain a common statement (decision, transacation value, outcome).
+This mechanism effectively brings the system to an irreversible state — meaning that a large potion of the network accepted a decision or a statement.
+So any such conflicting to that statement would be accepted with a negligible probability ε or smaller.
```

Comment:
No, it is not used anywhere else.
This is the often symbol for probability in cryptography, some consensus description and for number approaching.
That is why I used it here, it was used in the white paper as well

Reply:
IMO naming it `ε` makes perfect sense, but only if it is mentioned again somewhere. To me, even in documentation, declaring a variable but never using it is not a good practice, especially since the reader might expect it to come up later and go through the effort of keeping it in mind for nothing 😄 WDYT?

Reply:
Yes, makes sense. I realize I did it because of some unspecific plan to use it somewhere in the future, because I need to add a lot of other things - Stellar + comparison that are very likely to have it.
Do you mean to remove it now and add it later if it is used ?

Reply:
I don't know, for me if it isn't used within the page, it shouldn't be defined 😅 Even if it will later on be referenced in another page. Mentioning that the probability is referenced to as `ε` does not bring any value if we don't show any expression where that variable is used IMO 🤷 And if it's used in another document, that other document will most likely define what the `ε` variable signifies/represents.

Reply:
I was not sure if it was going to be used, because I haven't added the part of Avalanche that deals with analysis of its performance, which might reference it.
But also I understood that after this is merged Max will have a PR on it about removing all notations in order to simplify the text, and in that case I am not sure if that would stay also.
So my answer is that i am really not sure

Reply:
@AnNiran Agreed